cfgov/unprocessed/js/modules/util/assign.js

```diff
@@ -9,10 +9,10 @@
 
 'use strict';
 
-var fnBind = require( './fnBind' ).fnBind;
+var fnBind = require( './fn-bind' ).fnBind;
 
 /**
-* @param {Object} object - JavaScript object.
+* @param {object} object - JavaScript object.
```

Comment:
This one was good capitalized. The only lowercase ones would be `string`, `boolean`, `number`, `null`, and `undefined`